frontend/src/app/accountCreation/MfaVerify/MfaVerify.tsx

```diff
@@ -9,9 +9,11 @@ import StepIndicator from "../../commonComponents/StepIndicator";
 import { accountCreationSteps } from "../../../config/constants";
 import { AccountCreationApi } from "../AccountCreationApiService";
 import Alert from "../../commonComponents/Alert";
+import { LoadingCard } from "../LoadingCard/LoadingCard";
 
 interface Props {
   hint: ReactElement;
+  hideResend?: boolean;
```

Comment:
This generalized component is used to verify the security code from all MFA options - but resend isn't needed for TOTP, so this prop hides it